calcom/cal.com
# feat: Add all team members

## What does this PR do?

Inside assignement part (unlockable for team's managed event), add a toggle that will add the assignement to all members of the team.

Fixes #10991

## Requirement/Documentation

Given design : 
![image](https://github.com/calcom/cal.com/assets/64666894/ae6857af-f638-4331-8c3d-eee82807f7c1)


## Type of change




## How should this be tested?

Make sure to update your db with the new migration file.
Then follow these steps : 
- Start your app
- Create a team
- Create a managed event with your team
- Play with the new feature in the assignement tab

## Mandatory Tasks


https://github.com/calcom/cal.com/assets/64666894/19e1475f-7500-4763-98ba-68945717cf75
